docs/source/tutorial_notebooks/pipline_datasets.ipynb

```diff
@@ -0,0 +1,305 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9337f001-5e7c-4141-a60c-5e99052aee3d",
+   "metadata": {},
+   "source": [
+    "<img src=\"../_static/DREGS_logo.png\" width=\"300\"/>\n",
+    "\n",
+    "# Working with pipeline datasets\n",
+    "\n",
+    "This tutorial focuses on how to register data into `DREGS` from a\n",
+    "complete end-to-end pipeline. A \"pipeline\" in this context is any collection of\n",
+    "datasets that are inter-dependent, i.e., the output data from one process feeds\n",
+    "into the next process as its starting point. For example, a pipeline could\n",
+    "start with some raw imagery from a telescope, this raw imagery is then reduced\n",
+    "and fed into a piece of software that outputs a human-friendly value added\n",
+    "catalog. Or, a pipeline could be from a numerical simulation, starting with the\n",
+    "simulation's initial conditions, which then feed into an N-body code, which\n",
+    "then feed into a structure finder and gets reduced to a halo catalog.\n",
+    "\n",
+    "In the DESC data registry nomenclature, each stage of a pipeline is an\n",
+    "\"**execution**\", the data product(s) produced during each execution are \"**datasets**\",\n",
+    "and executions are linked to one another via \"**dependencies**\".\n",
+    "\n",
+    "### What we cover in this tutorial\n",
+    "\n",
+    "In this tutorial we will learn how to:\n",
+    "\n",
+    "- Register a series of dependant datasets from a pipeline into DREGS.\n",
+    "\n",
+    "### Before we begin\n",
+    "\n",
+    "If you haven't done so already, check out the [getting setup](file:///home/mcalpine/Documents/dataregistry/dataregistry/docs/build/html/tutorial_setup.html) page from the docs if you want to run this tutorial interactively.\n",
```

Comment:
Same comment as above: this appears to be referencing a file on your local machine.